src/dom-utils/getParentNode.js

```diff
@@ -7,10 +7,9 @@ export default function getParentNode(element: Node | ShadowRoot): Node {
   }
 
   return (
+    element.assignedSlot || // step into the shadow DOM of the parent of a slotted node
     element.parentNode || // DOM Element detected
     // $FlowFixMe: need a better way to handle this...
-    element.host || // ShadowRoot detected
-    document.ownerDocument || // Fallback to ownerDocument if available
-    document.documentElement // Or to documentElement if everything else fails
+    element.host // ShadowRoot detected
```

Comment:
```suggestion
    element.host || // ShadowRoot detected
    getDocumentElement(element)
```

I think this is what @atomiks suggested